master/internal/sproto/resources.go

```diff
@@ -73,6 +73,8 @@ type ResourcesStarted struct {
 	Addresses []cproto.Address
 	// NativeResourcesID is the native Docker hex container ID of the Determined container.
 	NativeResourcesID string
+	// HPCJobID is the Slurm/PBS job ID.
+	HPCJobID string
```

Comment:
Here is the assignment to HPCJobID happening?

Reply:
It will be in this determined-ee PR: 
https://github.com/determined-ai/determined-ee/pull/502/commits/688b71d6873a32e92e9905c27f51f2284d04f8bd